src/components/MyProfile.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const MyProfile = () => {
+  const data = useSelector((state) => state.mission);
+  const dataRocket = useSelector((state) => state.rockets);
+  return (
+    <div className="myprofile">
+      <div className="missions">
+        <h3>Missions</h3>
+        {
+           data.mission.map((item) => (item.reserved !== undefined && item.reserved ? <p key={item.id}>{item.mission_name}</p> : ''))
+         }
+      </div>
+      <div className="rockets">
+        <h3>Rockets</h3>
+        {
+           dataRocket.rocket.map((item) => (item.reserved !== undefined && item.reserved ? <p key={item.id}>{item.rocket_name}</p> : ''))
+         }
+      </div>
+    </div>
+  );
+};
```

Comment:
- [ OPTIONAL] I have a suggestion to enhance the My Profile section. Adding the `Cancel reservation` and `Leave Mission` buttons to the lists would be a great improvement. When users click on these buttons, they should be able to conveniently dispatch the actions already utilized in the main Rockets and Missions sections :wink: 

_This enhancement would create a more cohesive and user-friendly experience, streamlining interactions and making it easier for users to manage their reservations and missions efficiently._